ZUGFeRD/TradeLineItem.cs

```diff
@@ -334,6 +334,13 @@ public IList<TradeAllowanceCharge> GetSpecifiedTradeAllowanceCharges()
             return this.SpecifiedTradeAllowanceCharges;
         } // !GetSpecifiedTradeAllowanceCharges()
 
+        /// <summary>
+        /// The value given here refers to the superior line. In this way, a hierarchy tree of invoice items can be mapped.

```

Comment:
is 'superior' the right word? Shouldn't it just be called parent line? Same in another comment above

Reply:
I just took the wording from the documentation and I thought that maybe it wasn't so bad because the variable itself is already called ParentLineID

Reply:
They are really calling it 'superior'. Oh well, then it's ok of course.